src/Elasticsearch.Net/Api/RequestParameters/RequestParameters.SnapshotLifecycleManagement.cs

```diff
@@ -36,6 +36,12 @@ public class ExecuteSnapshotLifecycleRequestParameters : RequestParameters<Execu
 		public override HttpMethod DefaultHttpMethod => HttpMethod.PUT;
 	}
 
+	///<summary>Request options for ExecuteRetention <para>https://www.elastic.co/guide/en/elasticsearch/reference/current/slm-api-execute-retention.html</para></summary>
+	public class ExecuteRetentionRequestParameters : RequestParameters<ExecuteRetentionRequestParameters>
```

Comment:
Should the type name(s) be `ExecuteSnapshotLifecycleRetention*` ? The shortened method names are fine, because they reflect what is in the spec and are namespaced on `SnapshotLifecycleManagement`, but with the types, these live in the root `Nest` namespace, which feels like we ought to be more explicit to avoid naming collisions now and in future.